csfieldguide/templates/interactives/2d-arrow-manipulations.html

```diff
@@ -0,0 +1,200 @@
+                      {% trans 'Matrix: Multiple Transformation Challenge II' %}</a>
+                    <a class="dropdown-item" id="matrix-single" href="{% url 'interactives:interactive' '2d-arrow-manipulations' %}?config=matrix-single&showIndex">
+                      {% trans 'Matrix: Single Matrix Challenge' %}</a>
+                  </div>
+                </div>
+              </div>
+          
+              <div id="modules">
+                <div id="coordinates">
+                  <div id="heading"><p>x</p><p>y</p><p>{% trans 'highlight' %}</p></div>
+                  <div node-id="c0">
+                    <input class="input-box" id="p0-input-x"/>
+                    <input class="input-box" id="p0-input-y"/>
+                    <input class="input-box" type="radio" name="highlight-radios" id="c0-check">
+                  </div>
+                  <div node-id="c1">
+                    <input class="input-box" id="p1-input-x"/>
+                    <input class="input-box" id="p1-input-y"/>
+                    <input class="input-box" type="radio" name="highlight-radios" id="c1-check">
+                  </div>
+                  <div node-id="c2">
+                    <input class="input-box" id="p2-input-x"/>
+                    <input class="input-box" id="p2-input-y"/>
+                    <input class="input-box" type="radio" name="highlight-radios" id="c2-check">
+                  </div>
+                  <div node-id="c3">
+                    <input class="input-box" id="p3-input-x"/>
+                    <input class="input-box" id="p3-input-y"/>
+                    <input class="input-box" type="radio" name="highlight-radios" id="c3-check">
+                  </div>
+                  <div node-id="c4">
+                    <input class="input-box" id="p4-input-x"/>
+                    <input class="input-box" id="p4-input-y"/>
+                    <input class="input-box" type="radio" name="highlight-radios" id="c4-check">
+                  </div>
+                  <div node-id="c5">
+                    <input class="input-box" id="p5-input-x"/>
+                    <input class="input-box" id="p5-input-y"/>
+                    <input class="input-box" type="radio" name="highlight-radios" id="c5-check">
+                  </div>
+                  <div node-id="c6">
+                    <input class="input-box" id="p6-input-x"/>
+                    <input class="input-box" id="p6-input-y"/>
+                    <input class="input-box" type="radio" name="highlight-radios" id="c6-check">
+                  </div>
+                </div>
+            
+                <div id="matrices">
+                  <div id="matrix-first-scale">
+                    <h2>{% trans 'Multiplication Matrix' %}</h2>
+            
+                    <!-- default value is identity matrix -->
```

Comment:
can we make the input boxes a little bigger so when we type in something like -0.7 for example we can see the whole number? 
![image](https://user-images.githubusercontent.com/16066822/55691109-8d58b400-59ee-11e9-8702-96a8447665fa.png)